images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -70,9 +76,17 @@ make_bazel_rc () {
     # point bazel at our http cache ...
     # NOTE our caches are versioned by all path segments up until the last two
     # IE PUT /foo/bar/baz/cas/asdf -> is in cache "/foo/bar/baz"
-    local cache_id="$(get_workspace),$(hash_toolchains)"
-    local cache_url="http://${CACHE_HOST}:${CACHE_PORT}/${cache_id}"
+    local cache_id
+    cache_id="$(get_workspace),$(hash_toolchains)"
+    local cache_url
+    cache_url="http://${CACHE_HOST}:${CACHE_PORT}/${cache_id}"
     echo "build --remote_http_cache=${cache_url}"
+    # specifically for bazel 0.15.0 we want to set this flag
```

Comment:
do we think a fix might be cherry-picked for 0.15.1? or should we do this for all bazel 0.15 patch releases?

Reply:
this is a temporary check, we expect a fix to be in 0.15.1 if the fix is confirmed. I thought this would be good enough for now and avoid the question of handling semver. The goal of this PR is to make it possible to start testing this fix using a canary job and a :latest-experimental image

Reply:
we probably could match semver in bash and match a specific range with some regex though :^)